### ISSUE TITLE
Separate Landing with real app.

### ISSUE THREAD

**Issue by rasyidf** [SkalskiP/make-sense#203]
**Is your feature request related to a problem? Please describe.**
I'm an inexperienced UX/UI designer, as I mainly doing backend and ML jobs, but having to always click start button to show the file drop box every time we have to start a new session is somehow unnecessary, in fact, aside from aesthetic, the animation doesn't provide anything valuable like content loading as the underlying dop box is already being loaded at the same time.

**Describe the solution you'd like**
Separate Landing page of make sense from the Real editor.

**Describe alternatives you've considered**
-

**Additional context**
-

**Comment by SkalskiP** [comment]
Hello @rasyidf  👋  ! Thank you so much for your interest in make-sense. It seems that great minds think alike. Few weeks ago I started working on make-sense beta. As one of the elements, I would like to make a landing page on https://www.makesense.ai and host an editor on https://www.makesense.ai/editor. I would like to use https://spline.design/ to add an interesting animation to the homepage. I don't have a complete idea how the site should look like yet, and I want to design it and only then get down to programming.  Perhaps you would like to help me design the UX/UI?

I'll add that task to the scope of make-sense beta project.

**Comment by rasyidf** [comment]
Hi @SkalskiP, It's nice to be part of this awesome project, I have some idea with some flow, and perhaps I can help you with that, but I don't think I can make it soon enough. I have 1 weeks till I'm fully recovered and reach my desktop to start design, maybe well discuss next week.

**Comment by SkalskiP** [comment]
@rasyidf no problem. Maybe we could set up some call and talk this week?

**Comment by SkalskiP** [comment]
What communication channel do you prefer? Is Google Meet ok?

**Comment by rasyidf** [comment]
that will do, we can discuss this more at twitter.

**Comment by rasyidf** [comment]
> I would like to use https://spline.design/ to add an interesting animation

Anyway, I visited spline and it was awesome, we can also imitate their design system (separation of landing, docs, and app, but on a single domain.)

**Comment by SkalskiP** [comment]
> Anyway, I visited spline and it was awesome, we can also imitate their design system (separation of landing, docs, and app, but on a single domain.)

I agree. That would be a perfect structure. As for the landing page and editor, things are clear to me. I need to explore how to deploy docs on https://www.makesense.ai/dosc

**Comment by rasyidf** [comment]
what hosting provider you're using? does it have subdomain management feature available? like docs.makesense.ai app.makesense.ai and just makesense.ai

**Comment by SkalskiP** [comment]
I'm using AWS, and it should be possible.
